test/mobile/lightweight_dispatch/lightweight_dispatch_ops.yaml

```diff
@@ -0,0 +1,60 @@
+- aten::Float.Scalar
+- aten::Int.Tensor
+- aten::ScalarImplicit
+- aten::__contains__.str_list
+- aten::__getitem__.t
+- aten::_upsample_bicubic2d_aa.vec
+- aten::_upsample_bilinear2d_aa.vec
+- aten::_upsample_nearest_exact1d.vec
+- aten::_upsample_nearest_exact2d.vec
+- aten::_upsample_nearest_exact3d.vec
+- aten::adaptive_avg_pool1d
+- aten::adaptive_avg_pool1daten::upsample_bilinear2d.vec
+- aten::adaptive_avg_pool2d
+- aten::adaptive_avg_pool3d
+- aten::add.Tensor
+- aten::add.int
+- aten::append.t
+- aten::conv2d
+- aten::copy_
+- aten::copy_.Tensor
+- aten::detach
+- aten::div.Tensor_mode
+- aten::div.out
+- aten::divide.Tensor
+- aten::empty.memory_format
+- aten::eq.int
+- aten::eq.str
+- aten::fill_.Tensor
+- aten::flatten.using_ints
+- aten::floor.float
+- aten::gradient.scalarint
+- aten::gradient.scalarrayint
+- aten::gt.int
+- aten::index.Tensor
+- aten::item
+- aten::len.t
+- aten::list.t
+- aten::mul.float
+- aten::ne.int
+- aten::ones
+- aten::ones_like
+- aten::relu
+- aten::scalar_tensor
+- aten::select.int
+- aten::size
+- aten::size.int
+- aten::slice.t
+- aten::sub.int
+- aten::tensor
+- aten::tensor.float
+- aten::tensor.int
+- aten::to.device
+- aten::upsample_bicubic2d.vec
+- aten::upsample_bilinear2d.vec
+- aten::upsample_linear1d.vec
+- aten::upsample_nearest1d.vec
+- aten::upsample_nearest2d.vec
+- aten::upsample_nearest3d.vec
+- aten::upsample_trilinear3d.vec
+- aten::zeros
```

Comment:
How do we maintain this list going forward?

Reply:
This is a dump of all the operators used in the test. It would be hard for someone without context to maintain it. I guess next step I can write a script to automatically dump all the operators from the tests. We have similar script already so this should be feasible

Reply:
Added script to dump all operators in test models into yaml file.